src/sentry/lint/engine.py

```diff
@@ -113,7 +123,7 @@ def js_lint(file_list=None):
     return has_errors
 
 
-PRETTIER_VERSION = "1.2.2"
+PRETTIER_VERSION = "1.5.3"
```

Comment:
1.7.4?

Reply:
Hmm doesn't look like we use this at all

Reply:
if this was working for you it means our " Prettier is out of date:" message isn't working, right?